vscode/src/toitExec.ts

```diff
@@ -3,32 +3,38 @@
 // found in the LICENSE file.
 
 import { basename } from "path";
-import { window as Window } from "vscode";
+import { OpenDialogOptions, window as Window } from "vscode";
 import { toitExecFile, toitExecFilePromise } from "./cli";
 import { Device } from "./device";
 import { } from "./deviceView";
 import { Context, ensureAuth, selectDevice } from "./utils";
 
 
-function currentFilePath(suffix: string): string {
+async function pickFile(dialogOptions: OpenDialogOptions): Promise<string | undefined> {
+  const fileURI = await Window.showOpenDialog(dialogOptions);
+  if (!fileURI) return;  // File selection prompt dismissed.
+
+  return fileURI[0].path;
+}
+
+async function executeFilePath(suffix: string, dialogOptions: OpenDialogOptions): Promise<string | undefined> {
```

Comment:
Not a fan of `executeFilePath`. Makes me think of "execute this file path".
Maybe:
- `getExecuteFilePath`
- `sourceFilePath`
?

Reply:
Renamed to `getExecuteFilePath`